packages/react-dom/src/server/ReactPartialRenderer.js

```diff
@@ -570,13 +582,9 @@ class ReactDOMServerRenderer {
       if (nextChild === null || nextChild === false) {
         return '';
       } else {
-        if (React.isValidElement(nextChild)) {
```

Comment:
Maybe flatten this to "else if" at the level above? Now that we have else's in both branches.

Reply:
Oops! Somehow missed this haha.